contract-map.json

```diff
@@ -1,4 +1,2123 @@
+      "logo": "ring.svg",
+      "erc20": true,
+      "symbol": "RING",
+      "decimals": 18
+    },
+    "0xA31B1767e09f842ECFd4bc471Fe44F830E3891AA": {
+      "name": "ROOBEE",
+      "logo": "ROOBEE.svg",
+      "erc20": true,
+      "symbol": "ROOBEE",
+      "decimals": 18
+    },
+    "0xC7e43A1c8E118aA2965F5EAbe0e718D83DB7A63C": {
+      "name": "ZCore Token",
+      "logo": "zcrt.svg",
+      "erc20": true,
+      "symbol": "ZCRT",
+      "decimals": 18
+    },
+    "0x08d32b0da63e2C3bcF8019c9c5d849d7a9d791e6": {
+      "name": "Dentacoin Token",
+      "logo": "dcn.svg",
+      "erc20": true,
+      "symbol": "DCN",
+      "decimals": 0
+    },
+    "0xe2f2a5C287993345a840Db3B0845fbC70f5935a5": {
+      "name": "mStable USD",
+      "logo": "mUSD.svg",
+      "erc20": true,
+      "symbol": "mUSD",
+      "decimals": 18
+    },
+    "0xa3BeD4E1c75D00fa6f4E5E6922DB7261B5E9AcD2": {
+      "name": "Meta",
+      "logo": "MTA.svg",
+      "erc20": true,
+      "symbol": "MTA",
+      "decimals": 18
+    },
+    "0xe3818504c1B32bF1557b16C238B2E01Fd3149C17": {
+      "name": "Pillar",
+      "logo": "plr.svg",
+      "erc20": true,
+      "symbol": "PLR",
+      "decimals": 18
+    },
+    "0xba100000625a3754423978a60c9317c58a424e3D": {
+      "name": "Balancer (BAL)",
+      "logo": "balancer.svg",
+      "erc20": true,
+      "symbol": "BAL",
+      "decimals": 18
+    },
+    "0x8a854288a5976036A725879164Ca3e91d30c6A1B": {
+      "name": "Guaranteed Entrance Token",
+      "logo": "get-logo.svg",
+      "erc20": true,
+      "symbol": "GET",
+      "decimals": 18
+    },
+    "0xc12d099be31567add4e4e4d0D45691C3F58f5663": {
+      "name": "Auctus",
+      "logo": "auc.svg",
+      "erc20": true,
+      "symbol": "AUC",
+      "decimals": 18
+     },
+      "0xdF574c24545E5FfEcb9a659c229253D4111d87e1": {
+      "name": "HUSD",
+      "logo": "HUSD.svg",
+      "erc20": true,
+      "symbol": "HUSD",
+      "decimals": 8
+    },
+    "0x2ecB13A8c458c379c4d9a7259e202De03c8F3D19": {
+      "name": "Block-Chain.com Token",
+      "logo": "block-chain.svg",
+      "erc20": true,
+      "symbol": "BC",
+      "decimals": 18
+    },
+    "0x38e4adB44ef08F22F5B5b76A8f0c2d0dCbE7DcA1": {
+      "name": "Concentrated Voting Power",
+      "logo": "PowerPool.svg",
+      "erc20": true,
+      "symbol": "CVP",
+      "decimals": 18
+    },
+    "0xa117000000f279D81A1D3cc75430fAA017FA5A2e": {
+      "name": "Aragon Network Token",
+      "logo": "antv2.svg",
+      "erc20": true,
+      "symbol": "ANT",
+      "decimals": 18
+    },
+    "0xDecade1c6Bf2cD9fb89aFad73e4a519C867adcF5": {
+      "name": "Experty Wisdom Token",
+      "logo": "wisdom-token.svg",
+      "erc20": true,
+      "symbol": "WIS",
+      "decimals": 18
+    },
+    "0x7D85e23014F84E6E21d5663aCD8751bEF3562352": {
+      "name": "Axion",
+      "logo": "axion.svg",
+      "erc20": true,
+      "symbol": "AXN",
+      "decimals": 18
+    }
+=======
```

Comment:
Probably you should remove this.

Reply:
thanks @xrn Im updating a few things and come back to you!

Reply:
Hi @xrn, everything should be fine now. There were duplicated assets from the commits that were lacking in my branch. Following the previous discussion we removed long name assets (uniswap market), also regarding the limitation we have one asset symbol which is stkAAVE (7 chars) but there is a restriction to 6 chars so we set it as stAAVE meanwhile, would be possible to add an exception over this asset to put his whole symbol 'stkAAVE'?